omnigan/generator.py

```diff
@@ -84,7 +90,7 @@ def __init__(self, opts):
 
         self.decoders = nn.ModuleDict(self.decoders)
 
-    def translate(self, batch, translator="f", z=None):
+    def translate_batch(self, batch, translator="f", z=None):
```

Comment:
updated api as `translate` was not clear enough to understand it expects a batch